armi/reactor/tests/test_zones.py

```diff
@@ -203,7 +203,11 @@ def test_createHotZones(self):
         # Create a single assembly zone to verify that it will not create a hot zone
         single = zones.Zone("single")
         daZones.add(single)
-        aLoc = r.core.getFirstAssembly(Flags.FUEL).getLocation()
+        aLoc = next(
+            a
+            for a in r.core.getAssemblies(Flags.FUEL)
+            if a.spatialLocator.getRingPos() == (1, 1)
+        ).getLocation()
```

Comment:
pretty sure there's a better way to get assemblies by ring and position. 

Hey that'd be a good gallery example... ;)

Reply:
You mean a Composite method like `getChildAtLocation(loc: grids.SpatialLocator)`? That would be a wonderful idea :P Closest thing we have is the rather peculiar `getLocationContents()` on the Core, which uses the to-be defunct location labels. Unless I'm missing something...